### PR TITLE
[WIP] month view implementation of the calendar

### PR DESCRIPTION
**Pull request by InnaAtanasova**

### CODE REVIEW

File: library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html [hunk 1]
```diff
@@ -1,3 +1,22 @@
-<p>
-  calendar2-month-view works!
-</p>
+<div class="fd-calendar__months">
+  <ul class="fd-calendar__list">
+      <li
+          class="fd-calendar__item"
+          *ngFor="let month of monthNames; let i = index"
+          [ngClass]="{
+              ' fd-calendar__item--current': i + monthOffset === currentMonth,
```
Comment (MattL75) [inline library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html]: remove extra space before fd-calendar

File: library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+              ' is-selected': i + monthOffset === monthSelected
```
Comment (MattL75) [inline library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html]: here too

File: library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+          }"
+          (click)="selectMonth(i, $event)"
+      >
```
Comment (MattL75) [inline library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html]: can we not put these on their own line? I know VSCode does it by itself but it's so ugly

File: library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html (hunk 1 above)
Comment (InnaAtanasova) [reply library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html]: done

File: library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html (hunk 2 above)
Comment (InnaAtanasova) [reply library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html]: done

File: library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html (hunk 3 above)
Comment (InnaAtanasova) [reply library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html]: done

**Comment by JKMarkowski** [comment]
Now it looks good for me.

**Comment by JKMarkowski** [comment]
There are some little things missing:
- Passing currentlyDisplayed to monthView
- Throwing event, when some month is displayed, on current calendar, we switch to day view, when the month is choosen.

**Pull request closed without merging**